common/scala/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/CosmosDBSupport.scala

```diff
@@ -17,13 +17,14 @@
 
 package org.apache.openwhisk.core.database.cosmosdb
 
+import akka.event.slf4j.SLF4JLogging
 import com.microsoft.azure.cosmosdb._
 import com.microsoft.azure.cosmosdb.rx.AsyncDocumentClient
 
 import scala.collection.JavaConverters._
 import scala.collection.immutable
 
-private[cosmosdb] trait CosmosDBSupport extends RxObservableImplicits with CosmosDBUtil {
+private[cosmosdb] trait CosmosDBSupport extends RxObservableImplicits with CosmosDBUtil with SLF4JLogging {
```

Comment:
Why a different logging methodology?

Reply:
This is a one time initialization flow which so far was not passing the `Logging` implicit. So thought its fine to use Slf4j directly. Can refactor to pass the `Logging`

Reply:
Besides consistency, is there a concern for passing `implicit` vs using `SLF4JLogging` ?